guides/common/modules/proc_propagating-scap-content-using-ansible-deployment.adoc

```diff
@@ -0,0 +1,37 @@
+[id="Propagating_SCAP_Content_using_Ansible_Deployment_{context}"]
+= Propagating SCAP Content using Ansible Deployment
+
+Using this procedure, you can promote Security Content Automation Protocol (SCAP) content through the load balancer in the scope of the Ansible deployment method.
+
+.Prerequisite
+* Ensure that you have configured {Project} for Ansible deployment of compliance policies.
+For more information, see {ManagingSecurityDocURL}configuring-compliance-policy-deployment-methods_security-compliance[Configuring Compliance Policy Deployment Methods] in _{ManagingSecurityDocTitle}_.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Variables*.
+. Search for the `foreman_scap_client_port` variable and click its name.
+. In the *Default Behavior* area, ensure that the *Override* checkbox is selected.
+. In the *Parameter Type* list, ensure that `integer` is selected.
+. In the *Default Value* field, enter `{smartproxy_port}`.
+. In the *Specify Matchers* area, remove all matchers that override the default value.
+. Click *Submit*.
+. Search for the `foreman_scap_client_server` variable and click its name.
+. In the *Default Behavior* area, ensure that the *Override* checkbox is selected.
+. In the *Parameter Type* list, ensure that `string` is selected.
+. In the *Default Value* field, enter the FQDN of your load balancer, such as `_{loadbalancer-example-com}_`.
```

Comment:
```suggestion
. In the *Default Value* field, enter the FQDN of your load balancer, for example `_{loadbalancer-example-com}_`.
```

Reply:
I'd prefer to keep it, because earlier, we were trying to replace "for example" with "such as".